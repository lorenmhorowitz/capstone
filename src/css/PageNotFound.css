.Img {
  background-image: url("https://cdn.discordapp.com/attachments/489154865344282625/508368921666256936/Office.png");
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  top: 0%;
  bottom: 0%;
  right: 0%;
  left: 0%;
  position: fixed;
  overflow-y: auto;
}

.loginSquare {
  background-color: rgba(38, 38, 38, 0.8);
  border: 2px solid #f96302;
  height: 50%;
  width: 35vw;
  min-height: 500px;
  min-width: 350px;
  margin: auto;
  margin-top: 15vh;
  text-align: center;
}

#pageNotFound {
  color: white;
  margin-top: 15%;
  text-align: center;
}

#returnToLogin {
  background-color: gray;
  color: white;
  margin-top: 5%;
  text-align: center;
}
