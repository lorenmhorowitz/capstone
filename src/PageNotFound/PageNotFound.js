import React from "react";
import "../css/Login.css";
import Typography from "@material-ui/core/Typography/Typography";

const PageNotFound = () => {
  return (
    <div className="Img">
      <div id="loginSquare">
        <Typography style={{ color: "white" }}>Page Not Found</Typography>
      </div>
    </div>
  );
};

export default PageNotFound;
