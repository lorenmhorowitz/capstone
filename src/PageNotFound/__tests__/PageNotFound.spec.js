import React from "react";
import PageNotFound from "../PageNotFound";
import { shallow } from "enzyme";

describe("Page Not Found ", () => {
  it("renders correctly", () => {
    const rendered = shallow(<PageNotFound />);

    expect(rendered).toMatchSnapshot();
  });
});
