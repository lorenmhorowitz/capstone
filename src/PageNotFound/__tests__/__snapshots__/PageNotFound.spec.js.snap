// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found  renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageNotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        id="loginSquare"
      >
        <WithStyles(Typography)
          style={
            Object {
              "color": "white",
            }
          }
        >
          Page Not Found
        </WithStyles(Typography)>
      </div>,
      "className": "Img",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <WithStyles(Typography)
          style={
            Object {
              "color": "white",
            }
          }
        >
          Page Not Found
        </WithStyles(Typography)>,
        "id": "loginSquare",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Page Not Found",
          "style": Object {
            "color": "white",
          },
        },
        "ref": null,
        "rendered": "Page Not Found",
        "type": [Function],
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          id="loginSquare"
        >
          <WithStyles(Typography)
            style={
              Object {
                "color": "white",
              }
            }
          >
            Page Not Found
          </WithStyles(Typography)>
        </div>,
        "className": "Img",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <WithStyles(Typography)
            style={
              Object {
                "color": "white",
              }
            }
          >
            Page Not Found
          </WithStyles(Typography)>,
          "id": "loginSquare",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Page Not Found",
            "style": Object {
              "color": "white",
            },
          },
          "ref": null,
          "rendered": "Page Not Found",
          "type": [Function],
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
